docs/reuse/common-blocks.md

```diff
@@ -0,0 +1,44 @@
+Start: Install Pebble binary
```

Comment:
Can we create these files with more specific names? I also wonder if it's more discoverable to have one file per snippet? Does the directory need to be called "reuse"? For example, I'm thinking:

```
docs/common/install.md
docs/common/verify.md
```